product_docs/docs/pgd/5/raft/01_raft_subgroups_and_tpa.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: "Creating Raft subgroups using TPA"
+---
+
+The `TPAexec configure` command will automatically enable Raft subgroups if the locations are set in active-locations.
+
+## Creating Raft subgroups using TPA
+
+In this example, a two location cluster with three data nodes in each location will be created. The nodes in each location will be part of a PGD Raft subgroup for the location. 
+
+The cluster's name is `pgdgroup` which will also be the name of the top-level group.
+
+The cluster's two locations will be `us_east` and `us_west`, respectively.
+
+Each location will have four nodes. Three data nodes and a barman backup node. The three data nodes will also co-host pgd proxy. The configuration can be visualized like so:
+
+![6 Node Cluster with 2 Raft Subgroups](images/6NodeRaftSubGroups.png)
+
+The barman nodes do not participate in the subgroup and by extension, not the Raft group. They are therefore not shown. This diagram is a snapshot of a potential state of the cluster with the West Raft group having selected west_1 as write leader and west_2 as its own Raft leader. Over on the East, east_1 is write leader while east_3 is Raft leader. The entire cluster is contained within the top-level Raft group and there, west_3 is currently Raft leader.
```

Comment:
Nice job of getting east and west correct. The explanation is clear and concise as well, IMO. However, the top-group Raft leader `west_3` seems to live more in the East DC. I recommend to center that, just like the title. By colors, it's in the top-level group, so that's correct. Just the placement irritated me on first sight.

Reply:
Graphic is corrected now

Reply:
Note, these are placeholder graphics and design will need to make them brand ready. I tried creating these with Mermaid, but unfortunately concentric intersecting boxes break its hierarchy of objects.